src/main/scala/uk/gov/nationalarchives/PuidRepository.scala

```diff
@@ -0,0 +1,48 @@
+package uk.gov.nationalarchives
+
+import cats.effect.kernel.Async
+import cats.implicits._
+import doobie.Transactor
+import doobie.implicits._
+import doobie.util.transactor.Transactor.Aux
+import uk.gov.nationalarchives.DatabaseConfig.DatabaseCredentials
+import uk.gov.nationalarchives.PuidRepository.{AllPuidInformation, PuidInformation}
+
+class PuidRepository[F[_]: Async](xa: Aux[F, Unit]) {
+
+  private def allowedPuids(): F[List[PuidInformation]] = {
+    val query = sql"""SELECT "PUID", 'NonJudgmentFormat'::text FROM "AllowedPuids"; """.query[PuidInformation].to[List]
+    query.transact(xa)
+  }
+
+  private def disAllowedPuids(): F[List[PuidInformation]] = {
```

Comment:
Maybe change the camel casing to: `disallowedPuids`?

Reply:
Disallowed always feels like two words to me :smile: I've changed it.